.github/workflows/build.yml

```diff
@@ -140,103 +140,122 @@ jobs:
         with:
           fetch-depth: 0
       - name: Build
-        run: docker build --target release -t $DOCKER_IMAGE_ID .
+        run: |
+          docker buildx create \
+            --name multibuilder \
+            --platform linux/amd64,linux/arm64 \
+            --bootstrap --use
+          docker buildx build \
+            --platform linux/amd64,linux/arm64 \
+            -t $DOCKER_IMAGE_ID .
       - name: Check
         run: |
+            docker buildx build --load -t $DOCKER_IMAGE_ID .
+            # Assert that simple cases works for the new built image
             docker run $DOCKER_IMAGE_ID version
             docker run $DOCKER_IMAGE_ID --help
-            docker run $DOCKER_IMAGE_ID help
-            docker run $DOCKER_IMAGE_ID run --help
-            docker run $DOCKER_IMAGE_ID inspect --help
-            docker run $DOCKER_IMAGE_ID status --help
-            docker run $DOCKER_IMAGE_ID stats --help
-            docker run $DOCKER_IMAGE_ID scale --help
-            docker run $DOCKER_IMAGE_ID pause --help
-            docker run $DOCKER_IMAGE_ID resume --help
```

Comment:
I am not against dropping some of those, but is there any reason it is done in this PR?

Reply:
It was my plan, I forgot to split it.